lib/cosmos/gui/qt_tool.rb

```diff
@@ -45,10 +45,37 @@ def initialize(options)
       @options = options
       @about_string = nil
 
+      base_class = self.class.to_s.split('::')[-1].downcase
+      config_dir = File.join(Cosmos::USERPATH, 'config', 'tools', base_class)
+      if File.exist? config_dir
+        @options.config_dir = config_dir
+        options.config_file = verify_config(@options.config_file, ".txt", base_class)
```

Comment:
config file support is scattered throughout so I consolidated it here. I would go through and remove the option from the rest of the tools so we have a consistent way of handling it. There are only a few tools which don't actually take a config file but would still see the option when requesting help at the command line with -h. It wouldn't hurt to pass a config file it would simply do nothing.